nyancat/v0.16/README.md

```diff
@@ -0,0 +1,67 @@
+# Nyancat Testnet V0.16
+
+To support `DeFi` functionalities, we introduced the [HTLC](https://stage.irisnet.org/docs/features/htlc.html) and [Coin Swap](https://stage.irisnet.org/docs/features/coinswap.html) modules in v0.16, so that investors can transfer their tokens across chains and trade them decentralized.
```

Comment:
..., so that users can swap tokens between different blockchains and trade them in a decentralized manner.